src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java

```diff
@@ -70,7 +73,7 @@ public void testRemoveFromCourseAction(){
         detailsPage.clickRemoveStudentFromCourse(courseId)
             .verifyStatus(Const.StatusMessages.INSTRUCTOR_REMOVED_FROM_COURSE);
         assertNull(BackDoor.getStudent(courseId, "AAMgtUiT.instr2@gmail.com"));
-    
+        detailsPage.verifyHtml("/adminAccountDetailsRemoveStudent.html");
```

Comment:
You can change this to verifyHtmlMainContent. I think one verifyHtml is enough for one  Ui test